pkg/api/2018-09-30-preview/default.go

```diff
@@ -0,0 +1,13 @@
+package v20180930preview
+
+func setDefaults(oc *OpenShiftManagedCluster) {
+	defaultRouterName := "default"
```

Comment:
default router name is `router`

Reply:
Hm, are you sure? I got this from https://github.com/openshift/openshift-azure/blob/a4338bc5aeaff48ba2a37bec28ae249231f47dd8/pkg/fakerp/fakerp.go#L275-L281

I'd be concerned if I'm setting the wrong name here while all the tests are passing...

Reply:
@ehashman you're right

Reply:
I thought this was referring to the actual router name instead of the routerprofile name.